apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ts

```diff
@@ -36,10 +36,11 @@ router.get(
         resLocals: res.locals,
         courseStaff,
         aiGradingEnabled,
-        aiGradingMode: res.locals.assessment_question.ai_grading_mode,
-        aiGradingStats: res.locals.assessment_question.ai_grading_mode
-          ? await calculateAiGradingStats(res.locals.assessment_question)
-          : null,
+        aiGradingMode: aiGradingEnabled && res.locals.assessment_question.ai_grading_mode,
+        aiGradingStats:
+          aiGradingEnabled && res.locals.assessment_question.ai_grading_mode
+            ? await calculateAiGradingStats(res.locals.assessment_question)
+            : null,
```

Comment:
Fix: if the feature flag is disabled, don't try to load data or pretend we're in AI grading mode.